sklearn/model_selection/_validation.py

```diff
@@ -343,6 +345,27 @@ def _normalize_score_results(scores, scaler_score_key="score"):
     return {scaler_score_key: scores}
 
 
+def _warn_about_fit_failures(results, error_score):
+    num_failed_fits = sum(result["fit_failed"] for result in results)
```

Comment:
Nit: This is using the falseness of `None`. I slightly prefer getting the errors first:

```python
fit_errors = [
        result["fit_error"] for result in results if result["fit_error"] is not None
    ]
if fit_errors:
    ...

```